src/Symfony/Component/Console/Tests/phpt/single_application/help_name.phpt

```diff
@@ -0,0 +1,35 @@
+--TEST--
+Single Application can be executed
+--ARGS--
+--help
+--FILE--
+<?php
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\SingleCommandApplication;
+
+$vendor = __DIR__;
+while (!file_exists($vendor.'/vendor')) {
+    $vendor = dirname($vendor);
+}
+require $vendor.'/vendor/autoload.php';
+
+(new SingleCommandApplication())
+    ->setName('My Super Command')
+    ->setCode(function (InputInterface $input, OutputInterface $output) {
```

Comment:
return value of the function should be `int` and the code must return 0.

Reply:
Good catch. Fixed